File: src/java/Boot/src/com/core/api/level/area/TickingAreasManager.java
package com.core.api.level.area;

public class TickingAreasManager {

    public static native boolean hasActiveAreas();
    public static native void addArea(int dimension,String name,int x,int y,int z,int range);
    public static native void addAreaB(int dimension, String name, int x,int y,int z, int xx,int yy,int zz);
    public static native void addEntityArea(int dimension, long uid);
    public static native int countStandaloneTickingAreas();
    public static native int countPendingAreas(int dimension);



}

File: src/dev/api/TickingAreasManager.js
let TickingAreasManagerJava = WRAP_JAVA("com.core.api.level.area.TickingAreasManager");
var TickingAreasManager = {
    hasActiveAreas(){
        return TickingAreasManagerJava.hasActiveAreas() == 1;
    },
    addArea(dimension, name, x,y,z,range){
        TickingAreasManagerJava.addArea(dimension,name,x,y,z,range)
    },
    addAreaPostions(dimension, name, x, y, z, xx, yy, zz){
        TickingAreasManagerJava.addAreaPostions(dimension, name, x, y, z, xx, yy, zz);
    },
    addEntityArea(dimension, ent){
        TickingAreasManagerJava.addEntityArea(dimension, ent);
    },
    countStandaloneTickingAreas(){
        return Number(TickingAreasManagerJava.countStandaloneTickingAreas());
    },
    countPendingAreas(dimension){
        return Number(TickingAreasManagerJava.countPendingAreas(dimension));
    }
};